async-gateway/service.go

```diff
@@ -81,14 +81,30 @@ func (s *service) GetWorkload(id string) (GetWorkloadResponse, error) {
 	log := s.logger.With(zap.String("id", id))
 
 	// download workload status
-	statusPath := fmt.Sprintf("%s/%s/status", prefix, id)
-	log.Debug("downloading status file", zap.String("path", statusPath))
-	statusBuf, err := s.storage.Download(statusPath)
+	statusPath := fmt.Sprintf("%s/%s/*", prefix, id)
```

Comment:
Nit: if this variable is only used for logging, it might be clearer to the reader to remove the variable and log a different variable which is used later in the code, or in-line the path in the log statement